packages/backend/test/core/StatusService.test.ts

```diff
@@ -9,7 +9,7 @@ describe('StatusService', () => {
       barService: new BarService(),
     })
 
-    expect(statusService.getStatus()).to.deep.eq({
+    expect<object>(statusService.getStatus()).toEqual({
```

Comment:
Use either `unknown` or `object` everywhere. I am not sure how i like the fact that this fails for infinite types.